src/Spd.Manager.Licence/BizProfileValidation.cs

```diff
@@ -0,0 +1,18 @@
+﻿using FluentValidation;
+
+namespace Spd.Manager.Licence;
+public class BizProfileUpdateRequestValidator : AbstractValidator<BizProfileUpdateRequest>
+{
+    public BizProfileUpdateRequestValidator()
+    {
+        RuleFor(r => r.BizTradeName).NotEmpty();
+        RuleFor(r => r.BizTypeCode).NotEmpty().IsInEnum();
+        RuleFor(r => r.BizBCAddress).NotEmpty();
+        RuleFor(r => r.BizMailingAddress)
+            .NotEmpty()
+            .When(r => r.MailingAddressIsSameBizAddress == false);
+        RuleFor(r => r)
+            .Must(r => r.BizMailingAddress?.AddressLine1 != r.BizBCAddress?.AddressLine1)
```

Comment:
what's this? it is possible that address line 1 is the same even though MailingAddressIsSameBizAddress = false.

Reply:
I removed the flag "MailingAddressIsSameBizAddress" and adjusted the way the validation is done.